UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -15,52 +16,48 @@ namespace HealthV2
 	/// </summary>
 	public partial class BodyPart : MonoBehaviour, IBodyPartDropDownOrgans
 	{
-		[NonSerialized]
-		[Tooltip("The Health Master associated with this part, will find from parents if not set in editor")]
-		protected LivingHealthMasterBase healthMaster = null;
+		public LivingHealthMasterBase HealthMaster {get; private set;}
 
-		public LivingHealthMasterBase HealthMaster
-		{
-			get { return healthMaster; }
-			set
-			{
-				healthMaster = value;
-				foreach (var bodyPart in ContainBodyParts)
-				{
-					SetUpBodyPart(bodyPart);
-				}
-				HealthMasterSet();
-			}
-		}
 
+		//TODO: prefab populator still contains bodyparts
 		/// <summary>
 		/// Storage container for things (usually other body parts) held within this body part
 		/// </summary>
 		[HorizontalLine]
 		[Tooltip("Things (eg other organs) held within this")]
-		public ItemStorage Storage = null;
+		public ItemStorage OrganStorage = null;
+
+		[NonSerialized]
+		public List<Organ> OrganList = new List<Organ>();
+
+		/// <summary>
+		/// Player sprites for rendering equipment and clothing on the body part container
+		/// </summary>
+		[Tooltip("Player sprites for rendering equipment and clothing on this")]
+		public PlayerSprites playerSprites;
+
+		[HideInInspector] public bool IsBleeding = false;
```

Comment:
```suggestion
		[NonSerialized]
		public bool IsBleeding = false;
```
Guessing we could make this  `[NonSerialized]`? Few others marked `[HideInInspector]` that could probably be treated the same, though no biggie.

Reply:
that should be a private bool, with a `IsBleeding => isBleeding`